opennlp-tools/lang/es/abb_ES.xml

```diff
@@ -0,0 +1,254 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+   Licensed to the Apache Software Foundation (ASF) under one
+   or more contributor license agreements.  See the NOTICE file
+   distributed with this work for additional information
+   regarding copyright ownership.  The ASF licenses this file
+   to you under the Apache License, Version 2.0 (the
+   "License"); you may not use this file except in compliance
+   with the License.  You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing,
+   software distributed under the License is distributed on an
+   "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+   KIND, either express or implied.  See the License for the
+   specific language governing permissions and limitations
+   under the License.
+-->
+
+<dictionary case_sensitive="false">
+  <entry>
+    <token>a.C.</token>
+  </entry>
+  <entry>
+    <token>a. de C.</token>
+  </entry>
+  <entry>
+    <token>a.J.C.</token>
+  </entry>
+  <entry>
+    <token>a. de J.C.</token>
+  </entry>
+  <entry>
+    <token>a. m.</token>
+  </entry>
+  <entry>
+    <token>apdo.</token>
+  </entry>
+  <entry>
+    <token>apdo.</token>
+  </entry>
+  <entry>
+    <token>aprox.</token>
+  </entry>
+  <entry>
+    <token>Av.</token>
+  </entry>
+  <entry>
+    <token>Avda.</token>
+  </entry>
+  <entry>
+    <token>Bs. As.</token>
+  </entry>
+  <entry>
+    <token>c.c.</token>
+  </entry>
+  <entry>
+    <token>cap.</token>
+  </entry>
+  <entry>
+    <token>D.</token>
+  </entry>
+  <entry>
+    <token>Da.</token>
+  </entry>
+  <entry>
+    <token>Dña.</token>
+  </entry>
+  <entry>
+    <token>d.C.</token>
+  </entry>
+  <entry>
+    <token>d. de C.</token>
+  </entry>
+  <entry>
+    <token>d.J.C.</token>
+  </entry>
+  <entry>
+    <token>d. de J.C</token>
```

Comment:
I am not sure, if this will actually be detected as an abbreviation because it doesn't end with a period.

Reply:
Good catch. I wonder if it will detect it as "después de jesuscristo" or "don de Jesus Cristo" :thinking:

Reply:
i.e. not sure if it goes in order, of tries the longest/best match, etc.

Reply:
It must not end with a dot. See other comment by Bruno with `n.°` from above.
This example obviously does not end with a dot.